plugins/retry/build.sbt

```diff
@@ -2,7 +2,7 @@ organization := "com.github.dcshock"
 
 name := "forklift-retry"
 
-version := "2.0"
+version := "2.1"
```

Comment:
This should probably be `2.2` since forklift already has a version `2.1`. We just haven't been bumping the version if there are no changes.